identifiers/country-do/README.md

```diff
@@ -0,0 +1,10 @@
+# OCD Division Identifier 
+
+This folder list the following administrative divisions of Dominican Republic :
+
+* Province [ref](https://es.wikipedia.org/wiki/C%C3%A1mara_de_Diputados_de_la_Rep%C3%BAblica_Dominicana) as province.
+* National District [ref](https://en.wikipedia.org/wiki/Distrito_Nacional) as state.
+* Electoral district [ref](https://es.wikipedia.org/wiki/C%C3%A1mara_de_Diputados_de_la_Rep%C3%BAblica_Dominicana) as ed.
```

Comment:
I noticed some provinces are divided into electoral districts but others are not. Do you have a source for the districts? I did a light search but didn't find anything two descriptive and didn't see on the linked wiki page where they're described.

Reply:
I didn't find a lot for resources. I relied on the [wikipedia page](https://es.wikipedia.org/wiki/C%C3%A1mara_de_Diputados_de_la_Rep%C3%BAblica_Dominicana) and the [official website](https://www.camaradediputados.gob.do/app/app_2011/cd_diputados_new.aspx).